cpp/arcticdb/processing/clause.cpp

```diff
@@ -326,34 +326,46 @@ Composite<EntityIds> ProjectClause::process(Composite<EntityIds>&& entity_ids) c
 }
 
 AggregationClause::AggregationClause(const std::string& grouping_column,
-                                     const std::unordered_map<std::string,
-                                     std::string>& aggregations):
-        grouping_column_(grouping_column),
-        aggregation_map_(aggregations) {
+                                     const std::unordered_map<std::string, std::variant<std::string, std::pair<std::string, std::string>>>& aggregations):
```

Comment:
This is reaching the point where I would consider using a structure or alias for the variant aggregation identifier